src/config/configUser.js

```diff
@@ -0,0 +1,121 @@
+export const userFields = [
```

Comment:
If this configUser is included to codebase, add also a similar code-comment as listingFields have. 
I.e. introduction to the structure of the config and what are the possible values that the components can handle.